### PR TITLE
Add reference to Microsoft.ServiceHub.Framework

### PR DESCRIPTION
**Pull request by tmat**
Will be needed for using IServiceBroker.

### CODE REVIEW

File: eng/Versions.props [hunk 1]
```diff
@@ -159,8 +160,8 @@
     <MicrosoftVisualStudioTextManagerInterop120Version>12.0.30110</MicrosoftVisualStudioTextManagerInterop120Version>
     <MicrosoftVisualStudioTextManagerInterop121DesignTimeVersion>12.1.30328</MicrosoftVisualStudioTextManagerInterop121DesignTimeVersion>
     <MicrosoftVisualStudioTextManagerInterop160DesignTimeVersion>16.0.0</MicrosoftVisualStudioTextManagerInterop160DesignTimeVersion>
-    <MicrosoftVisualStudioThreadingAnalyzersVersion>16.4.16</MicrosoftVisualStudioThreadingAnalyzersVersion>
-    <MicrosoftVisualStudioThreadingVersion>16.4.16</MicrosoftVisualStudioThreadingVersion>
+    <MicrosoftVisualStudioThreadingAnalyzersVersion>16.6.13</MicrosoftVisualStudioThreadingAnalyzersVersion>
```
Comment (jasonmalinowski) [inline eng/Versions.props:163]: Does this need to go into master-vs-deps?

File: eng/Versions.props [hunk 2]
```diff
@@ -239,7 +240,7 @@
       create a test insertion in Visual Studio to validate.
     -->
     <NewtonsoftJsonVersion>12.0.2</NewtonsoftJsonVersion>
-    <StreamJsonRpcVersion>2.2.34</StreamJsonRpcVersion>
+    <StreamJsonRpcVersion>2.4.34</StreamJsonRpcVersion>
```
Comment (dibarbet) [inline eng/Versions.props:243]: could we do a test insertion for this?

File: eng/Versions.props (hunk 1 above)
Comment (tmat) [reply eng/Versions.props:163]: Why? 16.6 has been released, no?

File: eng/Versions.props (hunk 2 above)
Comment (tmat) [reply eng/Versions.props:243]: Sure.

File: eng/Versions.props (hunk 1 above)
Comment (jasonmalinowski) [reply eng/Versions.props:163]: Nope. :smile:

Comment (tmat) [reply eng/Versions.props:163]: Previews have. Do we require non-preview for master?

Comment (tmat) [reply eng/Versions.props:163]: Actually, to F5 Roslyn. You can build without VS entirely.

**Comment by tmat** [comment]
Moved to master-vs-deps: https://github.com/dotnet/roslyn/pull/43757

**Pull request closed without merging**